### PR TITLE
tslint config

### PR DESCRIPTION
**Pull request by tkmct**
Add tslint config and add lint check on ci

Before merge, we need to fix all the tslint errors.

### CODE REVIEW

**Comment by syuhei176** [comment]
@tkmct can you solve conflict?

**Comment by syuhei176** [comment]
How do we apply format by `prettier`?

**Comment by tkmct** [comment]
`tslint --fix`

File: package.json [hunk 1]
```diff
@@ -29,15 +29,21 @@
     "format": "lerna run format",
     "start": "node app",
     "docs:wallet": "typedoc --out ./docs ./packages/chamber-wallet/src",
-    "docs": "lerna run docs"
+    "docs": "lerna run docs",
+    "lint": "tslint 'packages/**/*.ts'",
```
Comment (syuhei176) [inline package.json]: @tkmct 
This path pattern will include `dist/*.ts`.

How about this?
```
    "lint": "tslint 'packages/**/src/*.ts' 'packages/**/test/*.ts'",
    "lint:ci": "tslint 'packages/**/src/*.ts' 'packages/**/test/*.ts'"
```

Comment (tkmct) [reply package.json]: Ah, thank you! that's the right way.

**Comment by tkmct** [comment]
#330 is the one for applying format.

**Pull request merged**